pkg/controller/che/che_controller.go

```diff
@@ -546,7 +546,20 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 	}
 
 	// create exec and view roles for CheCluster server and workspaces
-	role, err := deploy.SyncRoleToCluster(deployContext, "exec", []string{"pods/exec"}, []string{"*"})
+	execPolicyRule := []rbac.PolicyRule{
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"pods/exec",
+			},
+			Verbs:     []string{
+				"*",
+			},
+		},
+	}
+	role, err := deploy.SyncRoleToCluster(deployContext, "exec", execPolicyRule)
```

Comment:
pls. create a new function in role.go
```
SyncExecRoleToCluster(..)
```
and move
``` 
execPolicyRule := []rbac.PolicyRule{
		{
			APIGroups: []string{
				"",
			},
			Resources: []string{
				"pods/exec",
			},
			Verbs:     []string{
				"*",
			},
		},
	}
	role, err := deploy.SyncRoleToCluster(deployContext, "exec", execPolicyRule)
```

Pls. do accordingly for other changes...

Reply:
Updated. @tolusha could you pls another look
Also still have some problems with **Check nightly "opm" bundle** action. From the logs i see a message:

> Nightly bundle file deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml should be updated in your pr

but che-operator/olm/update-nightly-bundle.sh script doesn't provide any updates.